main/js/api.js

```diff
@@ -61,14 +66,26 @@ async function startUpdate() {
     const status = document.getElementById('status').value;
     const candidateIds = document.getElementById('candidateIds').value.split(',');
     const apiInstance = document.getElementById('apiInstance').value;
+    const requestTotal = candidateIds.length;
+    let requestCount = 0;
+    document.getElementById('output-progress').style.visibility = 'visible';
+    document.getElementById('stop').disabled = false;
```

Comment:
UX: progress tracking isn't visible until the progress is started.

Reply:
See comment regarding visibility above